### ISSUE TITLE
Merge Restored Textscreen Music and add music to chapter narrations

### ISSUE THREAD

**Issue by skellytz** [SpellholdStudios/BGT-WeiDU#24]
[Restored Textscreen Music](http://www.shsforums.net/files/file/74-restored-textscreen-music-core-installation-package-9/) should be added as a component of the BGT package. Music (MUS: "Chapter") should also be mixed in with the chapter narrations. (Andrea C. and Salk)

**Comment by GwendolyneFreddy** [comment]
Good idea. That's what I have scheduled to do with all 1PP mods... All related mods into one single package!
